qingstor/sdk/build.py

```diff
@@ -45,7 +45,10 @@ def __repr__(self):
     def parse(self):
         parsed_operation = dict()
         parsed_operation["Method"] = self.operation["Method"]
-        parsed_operation["URI"] = self.parse_request_uri()
+        if self.config.enable_virtual_host_style:
+            parsed_operation["URI"] = self.parse_request_virtual_style_uri()
+        else:
+            parsed_operation["URI"] = self.parse_request_uri()
```

Comment:
I prefer `parse_request_virtual_host_style_uri` and `parse_request_path_style_uri`